### PR TITLE
Fixed issue about tag capitalization

### LINKED ISSUE #18209: Tests don't check tags' case sensitivity in "Basic HTML and HTML5: Inform with the Paragraph Element"
#### Describe your problem and - if possible - how to reproduce it
If you use uppercase `P` tags in this challenge, tests will output:
> Make sure your p element has a closing tag.

That took me a good few minutes of tinkering with the code here and there, refreshing the page and trying different placeholder texts to realize that I accidentally typed an uppercase `<P>` in the opening tag and autocompleted that with `TAB` to `</P>`.

Solution: either adjust autocompletion to convert tags to lowercase or include check for case sensitivity of tags in tests.

#### Add a Link to the page with the problem
[Basic HTML and HTML5: Inform with the Paragraph Element](https://learn.freecodecamp.org/responsive-web-design/basic-html-and-html5/inform-with-the-paragraph-element/)

#### If possible, add a screenshot here

error

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

Closes #18209

@scissorsneedfoodtoo , #19747

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@thecodingaviator, thank you for taking the time to create a new PR for this. But it's still not triggering the build for some reason! Maybe it has something to do with the commit history? @raisedadead, do you know what's going on here?

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inform-with-the-paragraph-element.english.md [hunk 1]
```diff
@@ -23,7 +25,7 @@ Create a <code>p</code> element below your <code>h2</code> element, and give it
 ```yml
 tests:
   - text: Create a <code>p</code> element.
-    testString: assert(($("p").length > 0), 'Create a <code>p</code> element.');
+    testString: assert(($("p").length > 0), 'Create a valid <code>p</code> element.');
```
Comment (lynxlynxlynx) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inform-with-the-paragraph-element.english.md]: Is it possible to create an invalid one?

Comment (thecodingaviator) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inform-with-the-paragraph-element.english.md]: Like this-
`<p Hello, This is Bot></p>`

**Comment by scissorsneedfoodtoo** [comment]
Sorry for the delay, @thecodingaviator. Everything LGTM! Thank you for another great contribution.

**Pull request merged**